### PR TITLE
Rooms page

### PR DESCRIPTION
**Pull request by SergeyShynkarenko**

### CODE REVIEW

File: app/scss/pages/_rooms.scss [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+					width: 900px;
+					height: 550px;
+				  background-repeat: no-repeat;
+				  background-position: center center;
+				  background-size: cover;
+				  @media (max-width : 640px) {
+						height: 500px;
+					}
+					@media (max-width : 480px) {
+						height: 400px;
```
Comment (IShinkarenko) [inline app/scss/pages/_rooms.scss]: too large height - set `250px`

File: app/scss/pages/_rooms.scss [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+			text-align: center;
+			@media (max-width : 550px) {
+				font-size: 15px;
+			}
+		}
+	}
+	&__card {
+		border: 2px solid #14274A;
+		margin-top: 50px;
+		.card {
```
Comment (IShinkarenko) [inline app/scss/pages/_rooms.scss]: move to new line

**always make sure there is no nesting!!!**

**if you have nesting something you do incorrect - or class naming or scss styles!!!**

File: app/rooms.html [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+						<span class="hamburger-inner"></span>
+					</span>
+				</a>
+				<a  class="header__logo" href="#">
+					<img src="images/dest/logo.svg" alt="Luxury Hotels">
+				</a>
+				<nav>
+					<ul class="menu__list">
+						<li class="menu__list-item">
+							<a class="menu__list-link" href="#">Home</a>
```
Comment (IShinkarenko) [inline app/rooms.html]: Add link to Home page

File: app/rooms.html [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+						</li>
+						<li class="menu__list-item">
+							<a class="menu__list-link" href="#">Facilities</a>
+						</li>
+						<li class="menu__list-item">
+							<a class="menu__list-link active" href="#">Rooms</a>
+						</li>
```
Comment (IShinkarenko) [inline app/rooms.html:36]: Add link to rooms page

File: app/scss/pages/_rooms.scss [hunk 5]
```diff
@@ -0,0 +1,149 @@
+.room {
+	&__cards {
+		padding: 50px 0;
+		&-title {
+			text-align: center;
+			font-size: 50px;
```
Comment (IShinkarenko) [inline app/scss/pages/_rooms.scss]: too large - make less

Made all headers font sizes less on 5px

File: app/scss/pages/_rooms.scss [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+					padding: 40px 15px;
+				}
+				@media (max-width : 760px) {
+					flex-direction: column;
+					align-items: center;
+				}
+				@media (max-width : 320px) {
+					padding: 15px 15px;
+				}
+				.details {
```
Comment (IShinkarenko) [inline app/scss/pages/_rooms.scss]: add cursor pointer

**Review by IShinkarenko — APPROVED** [review]

File: app/scss/pages/_rooms.scss (hunk 1 above)
Comment (SergeyShynkarenko) [reply app/scss/pages/_rooms.scss]: done

File: app/scss/pages/_rooms.scss (hunk 2 above)
Comment (SergeyShynkarenko) [reply app/scss/pages/_rooms.scss]: done

File: app/rooms.html (hunk 4 above)
Comment (SergeyShynkarenko) [reply app/rooms.html:36]: done

File: app/rooms.html (hunk 3 above)
Comment (SergeyShynkarenko) [reply app/rooms.html]: done

File: app/scss/pages/_rooms.scss (hunk 6 above)
Comment (SergeyShynkarenko) [reply app/scss/pages/_rooms.scss]: done

**Review by IShinkarenko — APPROVED** [review]

**Pull request merged**